jobs/scripts/nightly-builds/nightly-builds.sh

```diff
@@ -9,11 +9,21 @@ artifact()
 # if anything fails, we'll abort
 set -e
 
+BUILDREQUIRES="libaio-devel librdmacm-devel libattr-devel libxml2-devel readline-devel openssl-devel libibverbs-devel fuse-devel glib2-devel userspace-rcu-devel libacl-devel sqlite-devel libuuid-devel"
+
+if [ "${CENTOS_VERSION}" -eq 8 ]
+then
+    ENABLE_REPOS="--enablerepo=PowerTools,Devel,epel-release"
```

Comment:
`epel-release` is the name of a package, this will not make a difference. As you mentioned there is no need to install epel-release, so do not do that

Reply:
done